src/Core/Checkout/Cart/SalesChannel/CartOrderRoute.php

```diff
@@ -156,4 +157,23 @@ private function addAffiliateTracking(Cart $cart, DataBag $data): void
             $cart->setCampaignCode($campaignCode);
         }
     }
+
+    public function place(Cart $cart, SalesChannelContext $context, RequestDataBag $data): OrderPlaceResult
```

Comment:
You always told us to not add call public methods within decoratable classes 😉 
Why is this public?

Reply:
sorry was not my intention ;)